kratos/processes/tetrahedral_mesh_orientation_check.cpp

```diff
@@ -0,0 +1,267 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:           BSD License
+//                          Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//                   Riccardo Rossi
+//
+//
+
+// System includes
+#include <unordered_map>
+#include <utility>
+
+// External includes
+
+// Project includes
+#include "processes/tetrahedral_mesh_orientation_check.h"
+#include "utilities/math_utils.h"
+#include "utilities/variable_utils.h"
+#include "includes/key_hash.h"
+
+namespace Kratos
+{
+KRATOS_CREATE_LOCAL_FLAG(TetrahedralMeshOrientationCheck,ASSIGN_NEIGHBOUR_ELEMENTS_TO_CONDITIONS, 0);
+KRATOS_CREATE_LOCAL_FLAG(TetrahedralMeshOrientationCheck,COMPUTE_NODAL_NORMALS, 1);
+KRATOS_CREATE_LOCAL_FLAG(TetrahedralMeshOrientationCheck,COMPUTE_CONDITION_NORMALS, 2);
+KRATOS_CREATE_LOCAL_FLAG(TetrahedralMeshOrientationCheck,MAKE_VOLUMES_POSITIVE, 3);
+KRATOS_CREATE_LOCAL_FLAG(TetrahedralMeshOrientationCheck,ALLOW_CONDITIONS_WITH_SAME_GEOMETRY, 4);
```

Comment:
actualy i would call this ALLOW_REPEATED_CONDITIONS (and set it to no by default)

Reply:
OK

Reply:
This is done, look at the Outdated flag